pkg/arvo/lib/pull-hook.hoon

```diff
@@ -77,7 +109,9 @@
     *[(list card) _^|(..on-init)]
   ::  +on-pull-nack: handle failed pull subscription
```

Comment:
I'd recommend moving these specialized arms to be _above_ the standard gall ones, so you can find them more quickly when reading top-to-bottom.